pkg/api/const.go

```diff
@@ -256,13 +256,11 @@ const (
 const (
 	// DefaultUseInstanceMetadata set to false as Azure Stack today doesn't support instance metadata service
 	DefaultAzureStackUseInstanceMetadata = false
-
 	// DefaultAzureStackAcceleratedNetworking set to false as Azure Stack today doesn't support accelerated networking
 	DefaultAzureStackAcceleratedNetworking = false
-
+	DefaultAzureStackAvailabilityProfile   = AvailabilitySet
```

Comment:
nit: Good to add a comment for consistency purpose.

Reply:
yes, please add a comment here to the exported const